### PR TITLE
fix the new filemenu carrot

### LINKED ISSUE #33909: [Bug]: Design issues with New file dialogue
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

* Upper right corner not rounded
* Hover effect not rounded
* Arrow misaligned (not centered)

image

### Steps to reproduce

1. Use NC25 Beta 4
2. Open Files app
3. Open New file dialogue ("+")

### Expected behavior

New design properly implemented

### Installation method

_No response_

### Operating system

_No response_

### PHP engine version

_No response_

### Web server

_No response_

### Database engine version

_No response_

### Is this bug present after an update or on a fresh install?

_No response_

### Are you using the Nextcloud Server Encryption module?

_No response_

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

_No response_

### List of activated Apps

```shell
--
```

### Nextcloud Signing status

_No response_

### Nextcloud Logs

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by szaimen**
Close https://github.com/nextcloud/server/issues/33909

| Before | After |
|---|---|
| image | image |

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: apps/files/css/files.css [hunk 1]
```diff
@@ -907,7 +907,7 @@ table.dragshadow td.size {
   /* Center triangle */
 }
 .newFileMenu::after {
-  left: 57px !important;
+  left: 61px !important;
```
Comment (CarlSchwan) [inline apps/files/css/files.css:910]: Need to be done in the apps/files/css/files.scss

**Review by CarlSchwan — APPROVED** [review]

File: apps/files/css/files.css (hunk 1 above)
Comment (szaimen) [reply apps/files/css/files.css:910]: jep, pushed already

**Review by skjnldsv — APPROVED** [review]
:carrot:

**Comment by szaimen** [comment]
CI failure unrelated

**Pull request merged**